mysql-rds-replicator-notls/Dockerfile

```diff
@@ -0,0 +1,33 @@
+## ref: https://hub.docker.com/r/pubnative/mysql-rds-replicator
+
+ARG MYSQL_VERSION="5.7.24"
+FROM mysql:${MYSQL_VERSION}
+
+LABEL maintainer="Eros Garcia Ponte <eros902002@googlemail.com>"
```

Comment:
Why so?

Reply:
Same as the original container (it's a copy). It isn't correct nonetheless and should be fixed in other containers too.